pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -32,12 +32,16 @@ import (
 
 	networkv1alpha1 "k8s.io/cloud-provider-gcp/crd/apis/network/v1alpha1"
 	networkclientset "k8s.io/cloud-provider-gcp/crd/client/network/clientset/versioned"
-	"k8s.io/cloud-provider-gcp/crd/client/network/clientset/versioned/typed/network/v1alpha1"
 	"k8s.io/cloud-provider-gcp/providers/gce"
 
 	controllersmetrics "k8s.io/component-base/metrics/prometheus/controllers"
 )
 
+const (
+	// GNPFinalizer - finalizer value placed on GNP objects by GNP Controller
+	GNPFinalizer = "networking.gke.io/ccm"
```

Comment:
can we have a more meaningful name?
`networking.gke.io/gnp-finalizer" or something that makes it clear from the name?

Reply:
sounds good. done.

Reply:
from the PoV of the user, they will want to see which process owns the finalizer, they are already aware of that the string is a finalizer given the context.